plugins/account_history_plugin/account_history_plugin.cpp

```diff
@@ -467,7 +467,7 @@ void account_history_plugin::plugin_startup()
    db.add_index<transaction_history_multi_index>();
 
    my->chain_plug->chain().applied_block.connect ([&impl = my](const chain::block_trace& trace) {
-      impl->applied_block(trace.block);
+      impl->applied_block(trace);
```

Comment:
I vote we add `explicit` to block.hpp `block_trace(const signed_block& s);` constructor.

Reply:
I agree, added.